### PR TITLE
make side panels init from js

### PR DESCRIPTION
**Pull request by kkrisstoff**
2nd PR for Toolbars / SidePanels changes saga 
- all init methods for side panels now in one place. It better to make close modules for components. 
- Move html for side panel elements from index.html to template string. Now we can init html templates and use variables inside

### CODE REVIEW

File: modules/UI/UI.js [hunk 1]
```diff
@@ -127,20 +128,13 @@ function promptDisplayName() {
...
-
-/**
- * Initialize toolbars.
+ * Initialize toolbars with side panels.
  */
 function setupToolbars() {
+    // Initialize toolbar buttons
     Toolbar.init(eventEmitter);
+    // Initialize side panels
+    SidePanels.init(eventEmitter);
```
Comment (kkrisstoff) [inline modules/UI/UI.js]: moved here all methods for init side panel elements

File: modules/UI/side_pannels/contactlist/ContactListView.js [hunk 2]
```diff
@@ -69,10 +79,19 @@ function getContactEl (id) {
...
         this.lockKey = 'roomLocked';
         this.unlockKey = 'roomUnlocked';
+    },
+
+    /**
+     * setup ContactList Model into ContactList View
+     *
+     * @param model
+     */
+    setup (model) {
```
Comment (kkrisstoff) [inline modules/UI/side_pannels/contactlist/ContactListView.js]: create separate method for setup model. We use this methods in different parts of application

File: modules/UI/side_pannels/settings/SettingsMenu.js [hunk 3]
```diff
@@ -1,10 +1,65 @@
-/* global $, APP, AJS, interfaceConfig, JitsiMeetJS */
+/* global $, _, APP, AJS, interfaceConfig, JitsiMeetJS */
 
 import UIUtil from "../../util/UIUtil";
 import UIEvents from "../../../../service/UI/UIEvents";
 import languages from "../../../../service/translation/languages";
 import Settings from '../../../settings/Settings';
 
+const sidePannelsContainerId = 'sideToolbarContainer';
+const compiledTpl = _.template(`
```
Comment (hristoterezov) [inline modules/UI/side_pannels/settings/SettingsMenu.js]: What is `_`?  Why do we need this?

Comment (hristoterezov) [inline modules/UI/side_pannels/settings/SettingsMenu.js]: Isn't it going to be the same as using string?

**Comment by hristoterezov** [comment]
The language select box looks strange:
screen shot 2016-11-01 at 12 52 38 pm

File: modules/UI/side_pannels/profile/Profile.js [hunk 4]
```diff
@@ -1,10 +1,45 @@
...
+        <div class="sideToolbarBlock first">
+            <label class="first" data-i18n="profile.setDisplayNameLabel">
+            </label>
+            <input type="text" id="setDisplayName"
+                data-i18n="[placeholder]settings.name" 
+                placeholder="Name">
+        </div>
+        <div class="sideToolbarBlock">
+            <label data-i18n="profile.setEmailLabel"></label>
+            <input type="text" id="setEmail" placeholder="Enter e-mail">
```
Comment (hristoterezov) [inline modules/UI/side_pannels/profile/Profile.js]: This is the current HTML for this input:
`<input data-i18n="[placeholder]profile.setEmailInput" type="text" id="setEmail">`

File: modules/UI/side_pannels/settings/SettingsMenu.js [hunk 5]
```diff
@@ -1,10 +1,65 @@
...
+    <div id="settings_container" class="sideToolbarContainer__inner">
+        <div class="title" data-i18n="settings.title"></div>
+        <form class="aui">
+            <div id="languagesSelectWrapper" class="first hide">
```
Comment (hristoterezov) [inline modules/UI/side_pannels/settings/SettingsMenu.js]: It should be `<div id="languagesSelectWrapper" class="sideToolbarBlock first hide">`

File: modules/UI/side_pannels/contactlist/ContactListView.js [hunk 6]
```diff
@@ -1,8 +1,18 @@
-/* global $, APP, interfaceConfig */
+/* global $, APP, interfaceConfig, _ */
 import Avatar from '../../avatar/Avatar';
 import UIEvents from '../../../../service/UI/UIEvents';
 import UIUtil from '../../util/UIUtil';
 
+const sidePannelsContainerId = 'sideToolbarContainer';
+const compiledTpl = _.template(`
+     <div id="contacts_container" class="sideToolbarContainer__inner">
+        <div class="title" data-i18n="contactlist"></div>
```
Comment (kkrisstoff) [inline modules/UI/side_pannels/contactlist/ContactListView.js]: fixed

File: modules/UI/side_pannels/profile/Profile.js [hunk 7]
```diff
@@ -1,10 +1,45 @@
...
+const sidePannelsContainerId = 'sideToolbarContainer';
+const compiledTpl = _.template(`
+    <div id="profile_container" class="sideToolbarContainer__inner">
+        <div class="title" data-i18n="profile.title"></div>
+        <div class="sideToolbarBlock first">
+            <label class="first" data-i18n="profile.setDisplayNameLabel">
+            </label>
+            <input type="text" id="setDisplayName"
+                data-i18n="[placeholder]settings.name" 
+                placeholder="Name">
```
Comment (kkrisstoff) [inline modules/UI/side_pannels/profile/Profile.js]: fixed

File: modules/UI/side_pannels/profile/Profile.js (hunk 4 above)
Comment (kkrisstoff) [inline modules/UI/side_pannels/profile/Profile.js]: fixed

File: modules/UI/side_pannels/settings/SettingsMenu.js (hunk 5 above)
Comment (kkrisstoff) [inline modules/UI/side_pannels/settings/SettingsMenu.js]: fixed

**Pull request merged**